project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
+                              :exclusions [org.clojure/clojure]]]
```

Comment:
ezbake requires VPN access doesn't it? I'd rather not make it a hard dep globally if it means having to have private access each time, or consumers of the source have to comment it out to make things work.